lua/table-nvim/keymaps.lua

```diff
@@ -9,22 +9,26 @@ local edit = require('table-nvim.edit')
 local set_keymaps = function(buf)
   local maps = conf.get_config().mappings
 
-  local opts = { noremap = true, buffer = buf }
-  local opts_expr = { noremap = true, buffer = buf, expr = true }
+  local function opts(desc)
+    return { noremap = true, buffer = buf, desc = desc }
+  end
+  local function opts_expr(desc)
+    return { noremap = true, buffer = buf, expr = true, desc = desc }
+  end
 
-  map({ 'n', 'i' }, maps.next, nav.next, opts_expr)
-  map({ 'n', 'i' }, maps.prev, nav.prev, opts_expr)
-  map('n', maps.insert_row_up, edit.insert_row_up, opts)
-  map('n', maps.insert_row_down, edit.insert_row_down, opts)
-  map('n', maps.move_row_up, edit.move_row_up, opts)
-  map('n', maps.move_row_down, edit.move_row_down, opts)
-  map('n', maps.insert_column_left, edit.insert_column_left, opts)
-  map('n', maps.insert_column_right, edit.insert_column_right, opts)
-  map('n', maps.move_column_left, edit.move_column_left, opts)
-  map('n', maps.move_column_right, edit.move_column_right, opts)
-  map('n', maps.insert_table, edit.insert_table, opts)
-  map('n', maps.insert_table_alt, edit.insert_table_alt, opts)
-  map('n', maps.delete_column, edit.delete_current_column, opts)
+  map({ 'n', 'i' }, maps.next, nav.next, opts_expr('Go to next cell'))
+  map({ 'n', 'i' }, maps.prev, nav.prev, opts_expr('Go to previous cell'))
+  map('n', maps.insert_row_up, edit.insert_row_up, opts('Insert a row above the current row'))
+  map('n', maps.insert_row_down, edit.insert_row_down, opts('Insert a row below the current row'))
+  map('n', maps.move_row_up, edit.move_row_up, opts('Move the current row up'))
+  map('n', maps.move_row_down, edit.move_row_down, opts('Move the current row down'))
+  map('n', maps.insert_column_left, edit.insert_column_left, opts('Insert a column to the left of current column'))
+  map('n', maps.insert_column_right, edit.insert_column_right, opts('Insert a column to the right of current column'))
+  map('n', maps.move_column_left, edit.move_column_left, opts('Move the current column to the left'))
+  map('n', maps.move_column_right, edit.move_column_right, opts('Move the current column to the right'))
```

Comment:
Hi @kirasok thank you for the PR.

I think these descriptions are too long for which-key, these won't look good when someone sets their keymaps alongside (not under a dedicated prefix) other keymaps, like below.

![image](https://github.com/user-attachments/assets/5f5cd948-97a7-48bd-b0d0-15d31aaac1eb)

So, we should shorten the descriptions to 3 - 4 words max. For the most part a description that matches the name of the mappings will work just fine, so for `maps.insert_row_up` description could be `Insert row up`. I don't think anyone who is configuring their keymaps will be confused about what this description means.

Reply:
Yes, you are right. Original descriptions were too long. I've made them shorter.